connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -148,10 +149,12 @@ private[kusto] object CslCommandsGenerator {
     val compressionFormat = additionalExportOptions.getOrElse("compressionType", "snappy")
     val namePrefix = s"${directory}part$partitionId"
     val sizeLimitOverride = additionalExportOptions.get("sizeLimit").map(size => s"sizeLimit=${size.toLong * 1024 * 1024} ,").getOrElse("")
+    val nativeParquetString = additionalExportOptions.get("useNativeParquetWriter")
+      .map(b => s"useNativeParquetWriter=$b, ").getOrElse(if (!supportNewParquetWriter) "useNativeParquetWriter=false, " else "")
 
     var command =
       s""".export async $compress to parquet ("${storageParameters.storageCredentials.map(getFullUrlFromParams).reduce((s, s1) => s + ",\"" + s1)})""" +
-        s""" with ($sizeLimitOverride namePrefix="$namePrefix", compressionType="$compressionFormat"$additionalOptionsString) <| $query"""
```

Comment:
Quick question: Should we add in the Source options that if we use the useNativeParquetWriter and use 2.x versions of spark then , the additional options of VectorizedReader needs to be set (or) the exports will fail with the DELTA_BYTE_ARRAY error